### ISSUE TITLE
Empty variable declaration block is not highlighted.

### ISSUE THREAD

**Issue by e-dard** [golang/lint#177]
Hi,

Whilst tidying up some code I came across this:

```go
// ...
var ()
// ...
```

Lint does not pick this up or suggest it's redundant. I'm not aware of a good reason for `var ()` existing in a codebase. 

Thoughts on this one?

**Comment by dsymonds** [comment]
It's hard to get excited. It's not something that will affect the godoc output, nor make the code look particularly odd. Do you see this happening a lot?

**Comment by zimmski** [comment]
I searched with the regex `var \(\s*\)` through my go directory I only found this in testdata files. I am wondering if there is a reason why this is even allowed as well as `import()` and `const()`?

**Comment by dsymonds** [comment]
I've had a think about it, and can't convince myself that a `var ()` is stylistically bad. It's vestigial, but that's about it.
